mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.HTTPAdaptiveStream.MixProject do
   use Mix.Project
 
-  @version "0.4.0"
+  @version "0.3.0"
```

Comment:
I assume you haven't rebased to master (or merged master into the current branch) yet?

Reply:
I merged master into current branch.